lib/gitlab/client/pipeline_schedules.rb

```diff
@@ -30,6 +30,18 @@ def pipeline_schedule(project, id)
       get("/projects/#{url_encode project}/pipeline_schedules/#{id}")
     end
 
+    # Get all pipelines triggered by a pipeline schedule
+    #
+    # @example
+    #   Gitlab.get_pipelines_by_pipeline_schedule(5, 3)
+    #
+    # @param  [Integer, String] project The ID or name of a project.
+    # @param  [Integer] id The ID of the pipeline schedule.
+    # @return  [Array<Gitlab::ObjectifiedHash>]
+    def get_pipelines_by_pipeline_schedule(project, id)
```

Comment:
Could we rename the method so that it doesn't contain a verb?

```suggestion
    def pipelines_by_pipeline_schedule(project, id)
```

Reply:
good idea. I applied you suggestion. Thanks.